monitoring/boot_config_linux.go

```diff
@@ -253,10 +262,18 @@ func parseKernelVersion(input string) (*KernelVersion, error) {
 			"invalid kernel version minor: %v, expected a number",
 			parts[2])
 	}
+	patch, err := strconv.Atoi(parts[3])
+	if err != nil {
+		return nil, trace.BadParameter(
```

Comment:
I would add the entire input string (that's being parsed) to all these error messages.